SSA/Experimental/Bits/Fast/BitStream.lean

```diff
@@ -272,16 +272,31 @@ end BitwiseOps
 /-! # Addition, Subtraction, Negation -/
 section Arith
 
-def addAux (x y : BitStream) : Nat → Bool × Bool
+def addAux' (x y : BitStream) : Nat → Bool × Bool
```

Comment:
Why is this function renamed?

Reply:
I kept around the old version in case anyone still wants to use the old version. The new version is called "addAux"